packages/quantic/force-app/main/default/lwc/quanticRadioButtonsGroup/quanticRadioButtonsGroup.js

```diff
@@ -0,0 +1,160 @@
+import {LightningElement, api, track} from 'lwc';
+
+/**
+ * The `QuanticRadioButtonsGroupOption` type defines the shape of an option
+ * that can be passed to the `QuanticRadioButtonsGroup` component.
+ * @typedef {Object} QuanticRadioButtonsGroupOption
+ * @property {String} value The value of the option.
+ * @property {String} [label] The label of the option.
+ * @property {String} [iconName] The name of the icon to be displayed.
+ * @property {String} [tooltip] The tooltip of the option.
+ */
+
+export default class QuanticRadioButtonsGroup extends LightningElement {
```

Comment:
Please add a documentation for this new component,
example: 
```suggestion
/**
 * The `QuanticRadioButtonGroup` displays ...
 * @category Internal
 * @example
 * <c-quantic-radio-buttons-group options...>
 * </c-quantic-radio-buttons-group>
 */
export default class QuanticRadioButtonsGroup extends LightningElement {
```

Reply:
Even if it's Internal?

Reply:
Yeah so we know it is internal x)